signoff/__init__.py

```diff
@@ -361,14 +380,20 @@ def main(action, uninstalled, signed_off, quiet, username, password, package,
     elif action == "signoff":  # sign-off packages
         for signoff_pkg, local_pkg in packages:
             warn_if_outdated(signoff_pkg, local_pkg)
+            warn_if_bad(signoff_pkg)
         if options.noconfirm or confirm("Sign off {}?".format(
                 click.style(" ".join(pkgbases), bold=True))):
             for signoff_pkg, local_pkg in packages:
-                session.signoff_package(signoff_pkg)
-                click.echo("Signed off {}.".format(signoff_pkg["pkgbase"]))
+                try:
+                    session.signoff_package(signoff_pkg)
+                except requests.exceptions.HTTPError as e:
+                    click.echo("Could not sign off {} ({})".format(signoff_pkg["pkgbase"], e))
```

Comment:
What does an example error message look like here? I'm not able to test this myself right now.

Reply:
It looks like

   Could not sign off <package> (403 Client Error: Forbidden for url: https://www.archlinux.org/packages/<repo>/x86_64/<package>/signoff/)

Reply:
Looks good to me!